packages/dds/tree/docs/wip/identifiers.md

```diff
@@ -0,0 +1,73 @@
+# SharedTree Node Identifiers
```

Comment:
I recommend using lazy returns to separate sentences within a paragraph. It makes leaving comments much easier since comments have line-granularity. If you'd want to do that for this doc that's fine, just a nice thing to do for the future.

Reply:
Oh, now I understand why markdown only renders newlines if there are two newlines in the source :) Thanks.